custom/icds_reports/tests/outputs/agg_ccs_record_2017-04-01_5_sorted.csv

```diff
@@ -1,323 +1,199 @@
 state_id,district_id,block_id,supervisor_id,awc_id,month,ccs_status,trimester,caste,disabled,minority,resident,valid_in_month,lactating,pregnant,thr_eligible,rations_21_plus_distributed,tetanus_complete,delivered_in_month,anc1_received_at_delivery,anc2_received_at_delivery,anc3_received_at_delivery,anc4_received_at_delivery,registration_trimester_at_delivery,using_ifa,ifa_consumed_last_seven_days,anemic_normal,anemic_moderate,anemic_severe,anemic_unknown,extra_meal,resting_during_pregnancy,bp1_complete,bp2_complete,bp3_complete,pnc_complete,trimester_2,trimester_3,postnatal,counsel_bp_vid,counsel_preparation,counsel_immediate_bf,counsel_fp_vid,counsel_immediate_conception,counsel_accessible_postpartum_fp,has_aadhar_id,aggregation_level,valid_all_registered_in_month,institutional_delivery_in_month,lactating_all,pregnant_all,valid_visits,expected_visits,state_is_test,district_is_test,block_is_test,supervisor_is_test,awc_is_test
-st1,d1,b1,s1,a1,2017-04-01,lactating,,obc,no,no,yes,2,2,0,2,0,0,0,0,0,0,0,,0,0,0,0,1,1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,1,5,2,0,2,0,0,2,0,0,0,0,0
-st1,d1,b1,s1,a1,2017-04-01,other,,obc,no,no,yes,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,16,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,5,0,0,0,0,0,6.24,0,0,0,0,0
-st1,d1,b1,s1,a1,2017-04-01,other,,obc,no,yes,yes,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,0,0,0.39,0,0,0,0,0
-st1,d1,b1,s1,a1,2017-04-01,other,,sc,no,no,yes,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,5,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,0,0,8.17,0,0,0,0,0
-st1,d1,b1,s1,a1,2017-04-01,other,,st,no,no,yes,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,0,0,0.39,0,0,0,0,0
+st1,d1,b1,s1,a1,2017-04-01,lactating,"",obc,no,no,yes,2,2,0,2,0,0,0,0,0,0,0,,0,0,0,0,1,1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,1,5,2,0,2,0,0,2,0,0,0,0,0

```

Comment:
why are so many rows deleted from this table?

Reply:
Hey these are the rows which does not satisfied the filters in ccs_monthly hence was not included in ccs_monthly there also deleted from agg_ccs record. majorly these are those which has ccs_Status='other' because its not pregnant(add is null) and its not lactating(add<6 months)